R/expand.R

```diff
@@ -0,0 +1,65 @@
+#' Create a data frame from all combinations of the inputs
+#'
+#' @description
+#' `vec_expand_grid()` is inspired by [expand.grid()]. Compared with
```

Comment:
Maybe add a small sentence that explainds what `vec_expand_grid()` and `expand.grid()` do?